### ISSUE TITLE
Confirm profile email message related error

### ISSUE THREAD

**Issue by xkopenreview** [openreview/openreview-web#85]
steps to reproduce:
1. login and go to profile edit page
2. add email of another user and click confirm
(error message will be shown like below)
![image](https://user-images.githubusercontent.com/60613434/87972596-2cdf7000-ca95-11ea-9f89-11f56e69c456.png)
3. clear new email input and enter a new email not associated with any user and click confirm button

expected behavior:
banner shows "A confirmation email has been sent to {email}"

current behavior:
banner shows "{existing user's id}A confirmation email has been sent to {email}". (it looks like this issue was inherited from existing UI)
another issue is  that user nav links dropdown is blocked by the banner message (specific to openreview-web)

![image](https://user-images.githubusercontent.com/60613434/87972403-d6723180-ca94-11ea-8881-13540cf1d2cc.png)
